array_methods_lab.js

```diff
@@ -0,0 +1,258 @@
+//
```

Comment:
I see that your still developing your skills in array methods. Hit me up and let me know if you would like to schedule some time to go over these questions. I know we've moved on to DOM but array methods are also important.